app/mailers/api/v1/rents_mailer.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    class RentsMailer < ApplicationMailer
+      def new_rent(rent)
```

Comment:
It's better when you use redis as intermediate to no pass complete model. Prefer passing the rent_id and do a find to found it ;)